src/sentry/models/project.py

```diff
@@ -260,6 +260,9 @@ class flags(TypedClassBitField):
         # This project has sent feedbacks
         has_feedbacks: bool
 
+        # This project has sent new feedback
```

Comment:
could we clarify in comment that this flag signifies that these feedbacks are from the new widget?